pkg/apis/storage/util/helpers.go

```diff
@@ -26,6 +26,10 @@ const IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-c
 // TODO: remove Beta when no longer used
 const BetaIsDefaultStorageClassAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
 
+// IsWindowsStorageClassAnnotation represents a StorageClass annotation that
```

Comment:
do we need changes in both util and v1?

Reply:
I checked /storage/v1/util and /storage/util and found they were used in the following cases. It seems like set it to v1 is good enough.

1. for /storage/v1/util
test/e2e/framework/pv
test/e2e/framework/statefulset
test/e2e/storage

2. for /storage/util
pkg/registry/storage/storageclass
plugin/pkg/admission/storage/storageclass/setdefault